orttraining/orttraining/core/session/training_session.cc

```diff
@@ -152,13 +152,28 @@ Status TrainingSession::ConfigureForTraining(
                                          config.distributed_config.horizontal_parallel_size,
                                          config.distributed_config.pipeline_parallel_size});
 
+  int32_t pipeline_stage_id = config.pipeline_config.has_value() ?
```

Comment:
const int32_t?

Reply:
Thanks. Updated. 

---
In reply to: [479841534](https://github.com/microsoft/onnxruntime/pull/4648#discussion_r479841534) [](ancestors = 479841534)